pkg/ddl/backfilling_operators.go

```diff
@@ -131,17 +130,19 @@ func getWriterMemSize(idxNum int) (uint64, error) {
 	return memSize, nil
 }
 
-func getMergeSortPartSize(concurrency int, idxNum int) (uint64, error) {
-	writerMemSize, err := getWriterMemSize(idxNum)
+func getMergeSortPartSize(workerCntLimit, idxNum int) (uint64, error) {
+	_, writerCnt := calculateReadWriteWorkerCnt(workerCntLimit)
+	writerMemSize, err := getWriterMemSize(writerCnt, idxNum)
 	if err != nil {
 		return 0, nil
 	}
-	return writerMemSize / uint64(concurrency) / 10, nil
+	return writerMemSize / uint64(workerCntLimit) / 10, nil
```

Comment:
@ywqzzy should use taskConcurrency? as this worker-cnt might > cpu-count

Reply:
Yes